data/json/itemgroups/ammo.json

```diff
@@ -166,7 +166,8 @@
       [ "8mm_fmj", 30 ],
       [ "8mm_hvp", 10 ],
       [ "8mm_inc", 20 ],
-      [ "8mm_jhp", 10 ]
+      [ "8mm_jhp", 10 ],
+      [ "12mm", 10 ]
```

Comment:
You need alphabetical ordering. A higher spawn rate would be acceptable, say 40?